src/locales/en/default.yml

```diff
@@ -0,0 +1,67 @@
+buttons:
+  next: Continue
+  cancel: Cancel
+  continue: Continue
+
+govuk:
+  serviceName:
+  backLink: Back
+  errorSummaryTitle: There is a problem
+  error: Error
+  warning: Warning
+  skipLink: Skip to main content
+  cookie:
+    cookieBanner:
+      title: "Cookies on GOV.UK One Login"
+      heading: "Cookies on GOV.UK One Login"
+      paragraph1: "We use some essential cookies to make this service work."
+      paragraph2: "We'd like to set additional cookies so we can remember your settings, understand how people use the service and make improvements."
+      buttonAcceptText: "Accept analytics cookies"
+      buttonRejectText: "Reject analytics cookies"
+      linkViewCookiesText: "View cookies"
+      cookieBannerHideLink: "Hide this message"
+      cookieBannerAccept:
+        paragraph1: "You've accepted additional cookies. You can "
+        paragraph2: " at any time."
+      cookieBannerReject:
+        paragraph1: "You've rejected additional cookies. You can "
+        paragraph2: " at any time."
+      changeCookiePreferencesLink: "change your cookie settings"
+    cookieText: "GOV.UK uses cookies to make the site simpler."
+    cookieLink: "#"
+    cookieLinkText: "Find out more about cookies"
+  phaseBanner:
+    tag: beta
+    content: This is a new service – your <a href="https://signin.account.gov.uk/contact-us" target="_blank" class="govuk-link" rel="noopener" target="_blank">feedback (opens in new tab)</a> will help us to improve it.
+  footerNavItems:
+    meta:
+      items:
+        - href: https://signin.account.gov.uk/accessibility-statement
+          text: Accessibility statement
+        - href: https://signin.account.gov.uk/cookies
+          text: Cookies
+        - href: https://signin.account.gov.uk/terms-and-conditions
+          text: Terms and conditions
+        - href: https://signin.account.gov.uk/privacy-notice
+          text: Privacy notice
+        - href: https://signin.account.gov.uk/contact-us
```

Comment:
Do we want to update this link in the PR or add it to the scope of https://govukverify.atlassian.net/browse/KIWI-1152 ?

Reply:
Good shout, yeah we should